package/gluon-web-admin/luasrc/lib/gluon/web/model/admin/remote.lua

```diff
@@ -43,12 +48,15 @@ local s = f_password:section(Section)
 
 local pw1 = s:option(Value, "pw1", translate("Password"))
 pw1.password = true
+pw1.datatype = 'minlength(12)'
 function pw1.cfgvalue()
 	return ''
 end
 
-local pw2 = s:option(Value, "pw2", translate("Confirmation"))
+local pw2 = s:option(Value, "pw2", translate("Confirmation"),
+	translate("Minimum") .. " " .. 12 .. " " .. translate("characters"))
```

Comment:
Always use `translatef` with format strings instead of splitting up sentences into multiple translation strings (sentences aren't constructed the same way in all languages). The English string should be "%u characters min.".